### PR TITLE
First Pull Request for Backend API issue - Scoreboard

### PR DESCRIPTION
**Pull request by trfoss**
I just wanted to get this first endpoint cleared before we start making the other endpoints with a similar structure.

### CODE REVIEW

File: backend/scoreboard/views.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+
+def test(request):
+	return JsonResponse({"test":"test"}, safe=False)
+
+def scores_week(request):
+	week_ago = datetime.now() - timedelta(days=7)
+	test = datetime.now() - timedelta(days=21) # need a test date that reaches far enough back in our regress.db test data
+	query_result = KattisScore.objects.filter(created_at__gte=week_ago).values('kattis_handle').annotate(points=Max('score') - Min('score'))
```
Comment (donaldong) [inline backend/scoreboard/views.py]: This is great! But it's still a bit different from what we need

If we print `query_result.query`, we get
```
SELECT "scoreboard_kattisscore"."kattis_handle_id", (MAX("scoreboard_kattisscore"."score") - MIN("scoreboard_kattisscore"."score")) AS "points" FROM "scoreboard_kattisscore" WHERE "scoreboard_kattisscore"."created_at" >= 2019-03-26 02:13:19.978302 GROUP BY "scoreboard_kattisscore"."kattis_handle_id"
```

Here we're selecting the `kattis_handle_id` which is just an ID value. We need to join it with `KattisHandle` so we can return the user handle.

Now the response looks like
```
[{"username": 11, "points": 100.0}, {"username": 12, "points": 94099.0}]
```

Comment (trfoss) [reply backend/scoreboard/views.py]: Sounds good, Ill get on that.

Comment (trfoss) [reply backend/scoreboard/views.py]: Does this look better:
```
SELECT "scoreboard_kattishandle"."handle", (MAX("scoreboard_kattisscore"."score") - MIN("scoreboard_kattisscore"."score")) AS "points" FROM "scoreboard_kattisscore" INNER JOIN "scoreboard_kattishandle" ON ("scoreboard_kattisscore"."kattis_handle_id" = "scoreboard_kattishandle"."id") WHERE "scoreboard_kattisscore"."created_at" >= 2019-03-12 03:55:16.964273 GROUP BY "scoreboard_kattishandle"."handle"
```

Comment (donaldong) [reply backend/scoreboard/views.py]: Yep! Looks good to me.

**Comment by donaldong** [comment]
Oh, also we need to remove the used code.

**Comment by trfoss** [comment]
Yeah that's fine with me. But next time I see you I may ask about what you did just cause I'd like to know.

**Comment by donaldong** [comment]
Yeah, maybe we can do it together on Wednesday?

**Comment by trfoss** [comment]
Yeah that sounds good, same time as last week?

**Comment by donaldong** [comment]
Yep! I just reserved a study room. It's the same place and same time as last week.

**Comment by trfoss** [comment]
Awesome! Thank you.

**Pull request merged**